test/PooltogetherGovernor.t.sol

```diff
@@ -979,3 +986,286 @@ contract CastVoteWithReasonAndParams is ProposalTest {
+        "withdrawReserve(address,address)", STAKE_PRIZE_POOL, POOLTOGETHER_TREASURY
+      )
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      proposals.targets(), proposals.values(), proposals.calldatas(), _description, FOR
+    );
+
+    // Assert the reserve has been withdrawn
+    uint256 newAmount = prizePool.reserveTotalSupply();
+    assertEq(newAmount, 0, "New amount is incorrect");
+
+    // Assert that the reserve rate has been changed
+    assertEq(
+      _newRate, configurableReserve.reserveRateMantissa(reserveSource), "New reserve is not correct"
+    );
+  }
+
+  function testFuzz_SetNumV3Winners(uint256 usdcNumWinners, uint256 daiNumWinners) public {
+    // Cannot be 0 address
+	// https://etherscan.io/address/0x3d9946190907ada8b70381b25c71eb9adf5f9b7b#code#F1#L54
+    vm.assume(usdcNumWinners > 0);
+    vm.assume(daiNumWinners > 0);
+
+    string memory _description =
+      "Increase the number of winners in the following V3 pools";
+    address USDC_PRIZE_STRATEGY = 0x3D9946190907aDa8b70381b25c71eB9adf5f9B7b;
+    address DAI_PRIZE_STRATEGY = 0x178969A87a78597d303C47198c66F68E8be67Dc2;
+
+    IV3MultipleWinners usdcStrategy = IV3MultipleWinners(USDC_PRIZE_STRATEGY);
+    uint256 existingUsdcWinners = usdcStrategy.numberOfWinners();
+    assertEq(existingUsdcWinners, 1);
+
+    IV3MultipleWinners daiStrategy = IV3MultipleWinners(DAI_PRIZE_STRATEGY);
+    uint256 existingDaiWinners = daiStrategy.numberOfWinners();
+    assertEq(existingDaiWinners, 1);
+
+    ProposalBuilder proposals = new ProposalBuilder();
+    proposals.add(
+      USDC_PRIZE_STRATEGY, 0, abi.encodeWithSignature("setNumberOfWinners(uint256)", usdcNumWinners)
+    );
+    proposals.add(
+      DAI_PRIZE_STRATEGY, 0, abi.encodeWithSignature("setNumberOfWinners(uint256)", daiNumWinners)
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      proposals.targets(), proposals.values(), proposals.calldatas(), _description, FOR
+    );
+
+	// Assert the number of DAI winners is correct
+    uint256 newDaiWinners = daiStrategy.numberOfWinners();
+    assertEq(newDaiWinners, daiNumWinners);
+
+	// Assert the number of USDC winners is correct
+    uint256 newUsdcWinners = usdcStrategy.numberOfWinners();
+    assertEq(newUsdcWinners, usdcNumWinners);
+  }
+
+  function test_SendUniswapPosition() public {
+    string memory _description = "Send Uniswap V3 position to controlled multisig";
+    IERC721 uniswapPositionManager = IERC721(UNISWAP_POSITION_CONTRACT);
+
+    uint256 balance = uniswapPositionManager.balanceOf(TIMELOCK);
+    assertEq(balance, 6);
+
+    uint256 safeBalance = uniswapPositionManager.balanceOf(POOLTOGETHER_UNISWAP_SAFE);
+    assertEq(safeBalance, 0);
+
+    ProposalBuilder proposals = new ProposalBuilder();
+    proposals.add(
+      UNISWAP_POSITION_CONTRACT,
+      0,
+      abi.encodeWithSignature(
+        "transferFrom(address,address,uint256)", TIMELOCK, POOLTOGETHER_UNISWAP_SAFE, 454_861
```

Comment:
Maybe overkill, but if it's easy enough, maybe we can use a seed and list of IDs to randomize which NFT we send?

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/15/commits/9c28b0198c35bb93c524d24ad05deff81b441db1